Blackjack/Controller/GameTableViewController.swift

```diff
@@ -10,105 +10,107 @@ import UIKit
-    
-    var playersMoney = 500
-    var blackjackGame = BlackjackBrain()
-    var playersHand = [card]()
-    var dealersHand = [card]()
-    
-    
+    var doubleTapGesture: UITapGestureRecognizer!
+    var swipeRightGesture: UISwipeGestureRecognizer!
+    var swipeDownGesture: UISwipeGestureRecognizer!
     
     var playerCardViews = [UIImageView]()
     var dealersCardViews = [UIImageView]()
     
+    var playersHand: [card] {
+        get {
+            return blackjackGame.getPlayersHand()
+        }
+    }
+    var dealersHand: [card] {
+        get {
+            return blackjackGame.getDealersHand()
+        }
+    }
+    
     
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        
+        doubleTapGesture = UITapGestureRecognizer(target: self, action: #selector(stay(_:)))
+        swipeDownGesture = UISwipeGestureRecognizer(target: self, action: #selector(hit(_:)))
+        swipeRightGesture = UISwipeGestureRecognizer(target: self, action: #selector(clear(_:)))
+        
+        doubleTapGesture.numberOfTapsRequired = 2
+        swipeDownGesture.direction = .down
+        swipeRightGesture.direction = .right
+        
+        self.view.addGestureRecognizer(doubleTapGesture)
+        self.view.addGestureRecognizer(swipeDownGesture)
+        self.view.addGestureRecognizer(swipeRightGesture)
+    
+        setupView()
+    }
+    
+    func setupView() {
         playersHandLabel.isHidden = true
         dealersHandLabel.isHidden = true
+        doubleTapGesture.isEnabled = false
+        swipeDownGesture.isEnabled = false
+        swipeRightGesture.isEnabled = false
         
         moneyLabel.text = getMoneyString(forInt: playersMoney)
         bettingSlider.value = bettingSlider.minimumValue
         bettingLabel.text = "Betting: $\(bettingSlider.value)"
         bettingSlider.maximumValue = Float(playersMoney)
-        
-        let hitGetstureRecgonizer = UISwipeGestureRecognizer(target: self, action: #selector(hit(sender:)))
-        
-        let stayGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(stay(sender:)))
-        
-        let endGestureRocognnizer = UISwipeGestureRecognizer(target: self, action: #selector(clear(sender:)))
-        
-        stayGestureRecognizer.numberOfTapsRequired = 2
-        
-        hitGetstureRecgonizer.direction = .down
-        endGestureRocognnizer.direction = .right
-        
-        
-        view.addGestureRecognizer(hitGetstureRecgonizer)
-        view.addGestureRecognizer(stayGestureRecognizer)
-        view.addGestureRecognizer(endGestureRocognnizer)
     }
     
     override func viewDidAppear(_ animated: Bool) {
         let instructionsVC = UIStoryboard.init(name: "Instructions", bundle: nil).instantiateInitialViewController() as! InstructionsViewController
         
-        if !userDefaults.bool(forKey: Literals.visitedBefore)
-        {
+        if !userDefaults.bool(forKey: Literals.visitedBefore) {
```

Comment:
I need to add Swift Lint so you don't have to keep fixing my new line braces lol